src/ekat/ekat_pack.hpp

```diff
@@ -10,6 +10,7 @@
 #include "ekat/ekat_type_traits.hpp"
 
 #include <Kokkos_Core.hpp>
+#include "spdlog/fmt/ostr.h"
```

Comment:
Is this added to `ekat_pack.hpp` for convenience? To make logging more seamless when using packs?

Reply:
Yes.  The spdlog docs say you have to include that file wherever you define a type's `operator <<` to get native support from spdlogs functions.